lib/matplotlib/tests/test_mlab.py

```diff
@@ -1131,7 +1131,7 @@ def test_detrend_str_linear_2d_slope_off_axis0(self):
         res = mlab.detrend(input, key='linear', axis=1)
         assert_allclose(res, targ, atol=self.atol)
 
-    def test_detrend_detrend_linear_1d_slope_off_axis1(self):
+    def test_detrend_detrend_linear_1d_slope_off_axis1_notranspose(self):
```

Comment:
Was there a particular need to change these test names?

Reply:
Yes - there's another test with this name, so before renaming this test could never be run!